### ISSUE TITLE
rustdoc: Search results have larger margins than before

### ISSUE THREAD

**Issue by camelid** [rust-lang/rust#82542]
### nightly (2021-02-25)

![image](https://user-images.githubusercontent.com/37223377/109246332-ec687300-7796-11eb-855d-5dd0e7ec60e0.png)

### beta (1.51.0-beta.3)

![image](https://user-images.githubusercontent.com/37223377/109246369-00ac7000-7797-11eb-85ad-9bfff8592fce.png)

**Comment by camelid** [comment]
Pretty sure this happened in the last day or two because otherwise I would have noticed.

Not sure if this change is intentional or not, but the only PR I could find that I think could've caused this is #82313, which doesn't seem like it should've significantly changed UI.

**Comment by jyn514** [comment]
This seems fine to me, or at least not a bug. I think minor css changes are to be expected when we update css library.

**Comment by camelid** [comment]
I don't know, it feels a little too far spaced apart to me.

**Comment by hyd-dev** [comment]
Looks like not only search results are affected?
<details><summary><a href="https://doc.rust-lang.org/nightly/std/#modules">https://doc.rust-lang.org/nightly/std/#modules</a></summary>

![nightly](https://user-images.githubusercontent.com/30410251/109292589-90205480-7865-11eb-817a-2622d828992c.png)
</details>
<details><summary><a href="https://doc.rust-lang.org/beta/std/#modules">https://doc.rust-lang.org/beta/std/#modules</a></summary>

![beta](https://user-images.githubusercontent.com/30410251/109292620-9d3d4380-7865-11eb-9e3c-cc7e75d1ef7b.png)
</details>

**Comment by hyd-dev** [comment]
Oh... https://github.com/rust-lang/rust/pull/82313#issue-576852770
> TL;DR: Nothing changes except a slight adjustment to line height.
